lib/util.js

```diff
@@ -239,11 +232,36 @@ function createRouter(opts) {
  */
 function initAndLogRequest(req, app) {
     req.headers = req.headers || {};
-    req.headers['x-request-id'] = req.headers['x-request-id'] || generateRequestId();
+    req.headers['x-request-id'] = req.headers['x-request-id'] || uuidv1();
     req.logger = app.logger.child({
         request_id: req.headers['x-request-id'],
         request: reqForLog(req, app.conf.log_header_whitelist)
     });
+    req.context = { app, reqId: req.headers['x-request-id'] };
```

Comment:
The `app` is already a part of a request according to the express docs.

Reply:
Indeed, good point, removed it.